app/models/journey.rb

```diff
@@ -1,6 +1,8 @@
 class Journey < ApplicationRecord
   include Enums
-  include Searchable
+  include PgSearch
+
+  after_save :update_steps
```

Comment:
Preco?

Reply:
step je naviazany na published_status od Journey, kebyze zmenis status pre Journey z PUBLISHED na DRAFT tak sa neupdatne tabulka pre search a budu dohladatelne steps pre DRAFT.

Reply:
Aha! A nestacilo by tam zavolat `update_pg_search_document` ? https://www.rubydoc.info/github/Casecommons/pg_search/PgSearch%2FMultisearchable:update_pg_search_document

Reply:
dobry napad, da sa to zavolat nad kazdym step, tak si to myslel?

Reply:
jj presne tak. A mozno iba ak sa zmeni ten status.